tests/playbooks/tests_ethtool_coalesce.yml

```
@@ -0,0 +1,91 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: lsrfeat1
+    type: veth
+  tasks:
+    - block:
+        - name: >-
+            TEST: I can create a profile without any coalescing option.
+          debug:
+            msg: "##################################################"
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                type: "{{ type }}"
+                autoconnect: no
+                ip:
+                  dhcp4: no
+                  auto6: no
+        - name: Get profile's coalescing options
+          command: "grep ^ETHTOOL_OPTS /etc/sysconfig/network-scripts/ifcfg-{{ interface }}"
+          register: no_coalesce
+          ignore_errors: yes
+        - name: "ASSERT: The profile does not contain coalescing options"
+          assert:
+            that: no_coalesce.stdout == ""
+        - name: >-
+            TEST: I can set rx-frames and tx-frames.
+          debug:
+            msg: "##################################################"
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                type: "{{ type }}"
+                autoconnect: no
+                ip:
+                  dhcp4: no
+                  auto6: no
+                ethtool:
+                  coalesce:
+                    rx_frames: 128
+                    tx_frames: 64
+        - name: Get profile's coalescing options
+          command: "grep ^ETHTOOL_OPTS /etc/sysconfig/network-scripts/ifcfg-{{ interface }}"
+          register: with_coalesce
+        - name: Assert coalesce options set in profile
+          assert:
+            that:
+              - >-
+                  'ETHTOOL_OPTS=\"-C eth4 rx-frames 128 tx-frames 64\"' ==
```

I guess this should  be:
```suggestion
                  'ETHTOOL_OPTS=\"-C lsrfeat1 rx-frames 128 tx-frames 64\"' ==
```

But actually, it would be nice to rename the interface to something like `lsrcoalesce1` to make it easier to trace it back to this test in case it appears in an error.

---

Ouch, yes good catch!